frontend/src/metabase/dashboard/components/AddCardSidebar/AddCardSidebar.unit.spec.tsx

```diff
@@ -124,7 +126,9 @@ async function setup({
   });
 
   renderWithProviders(
-    <AddCardSidebar onSelect={jest.fn()} onClose={jest.fn()} />,
+    <MockDashboardContext>
+      <AddCardSidebar onSelect={jest.fn()} onClose={jest.fn()} />
```

Comment:
The `QuestionPicker` is rendered inside this component and it now relies on the dashboard context, so I need to wrap it here.